scripts/create-sdcard-image-from-scratch.sh

```diff
@@ -73,6 +73,16 @@ python3 -c "from notebook.auth.security import set_password; set_password('$pass
 # fix for permission error
 sudo chown -R jetbot:jetbot ~/.local/share/
 
+# Install jupyter_clickable_image_widget
+echo "\e[42m Install jupyter_clickable_image_widget \e[0m"
+cd
+sudo apt-get install libssl1.0-dev
+git clone https://github.com/jaybdub/jupyter_clickable_image_widget
```

Comment:
I added the v0.1 tag, so this wont break if master of jupyter widget changes.

https://github.com/jaybdub/jupyter_clickable_image_widget/tags

Can you set to use this tag?